except_star.md

```diff
@@ -14,20 +14,54 @@ group of unrelated exceptions being propagated together.
 ## Motivation
 
 The interpreter is currently able to propagate at most one exception at a
-time. The chaining features introduced in PEP3134 [reference] link together
-exceptions that are related to each other as the cause or context, but there
-are situations where there are multiple unrelated exceptions that need to be
-propagated together as the stack unwinds. Several real world use cases are
-listed below.
+time. The chaining features introduced in
+[PEP 3134](https://www.python.org/dev/peps/pep-3134/) link together exceptions that are
+related to each other as the cause or context, but there are situations where
+multiple unrelated exceptions need to be propagated together as the stack
+unwinds. Several real world use cases are listed below.
+
+* Concurrent errors. The Trio library for async concurrency has a `MultiError`
+  exception type used to report a collection of errors raised in parallel
+  in different child tasks. Work on this PEP was initially motivated by the
+  difficulties in handling `MultiError`s, which are detailed in a design
+  document for an [improved version, `MultiError2`]([https://github.com/python-trio/trio/issues/611).
+  This document demonstrates how difficult it is to create an effective API
+  for reporting and handling multiple errors without the language changes we
+  are proposing.
```

Comment:
Shouldn't we also mention that asyncio has a similar use case (e.g. gather())?

Reply:
@1st1 Would you like to edit this paragraph along these lines?